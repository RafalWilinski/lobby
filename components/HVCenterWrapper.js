import React from "react";

export default ({ children }) => (
  <div
    style={{
      display: "flex",
      width: "100%",
      justifyContent: "center",
      alignItems: "center",
      height: "100vh"
    }}
  >
    {children}
  </div>
);
