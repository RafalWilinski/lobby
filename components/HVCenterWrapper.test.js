import React from "react";
import renderer from "react-test-renderer";
import HVCenterWrapper from "./HVCenterWrapper";

describe("HVCenterWrapper", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HVCenterWrapper />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
