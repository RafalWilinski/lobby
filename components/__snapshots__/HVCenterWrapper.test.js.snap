// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HVCenterWrapper renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100vh",
      "justifyContent": "center",
      "width": "100%",
    }
  }
/>
`;
